docs/installation/instructions.rst

```diff
@@ -1,10 +1,18 @@
 Installation Instructions
 =========================
 
+Supported Platforms
+-------------------
+
+Pulp has been tested to work on Ubuntu, Fedora, and CentOS. It should work on other operating systems
```

Comment:
I don't think of this as "Supported Platforms". I hope we don't get into "cerifying" distributions. What about this as an idea:

Pulp can run on any distribution that provides a the Python 3.5+ runtime environment, the necessary dependencies, and supporting services, e.g. a database. Pulp has been demonstrated to run on Ubuntu, Fedora, CentOS, Debian, and OS X, and likely runs in many more places.